content/platform/overview/advanced-concepts/index.md

```diff
@@ -17,125 +17,104 @@ aliases:
+Trails will still support simple properties like types, descriptions and thumbnails for search and grouping purposes.
 {{< /note >}}
 
-Events are registered into collections called _Assets_. These Assets might represent an individual thing, a class of things, or something more abstract like 'all Events for this day'. Some care should be taken in designing the scheme for this as it will aid in simplifying sharing policies.
+In the current platform, Events are registered into collections called _Assets_.
+[Assets](/developers/api-reference/assets-api/) may represent an individual thing, a class of things, or something more abstract like 'all Events for this day'.
+As Assets are retired, Events can still be correlated by Trails or Event Attributes.
 
-A simple Asset might look like this:
+Some care should be taken in designing the scheme for this as it will aid in simplifying sharing policies.
 
-```json
-   {
-      // Fixed global identity for this Asset
-      "identity": "assets/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-
-      // The set of tracked and traced attributes
-      "attributes": {
-        // arc_* are understood and processed by DataTrails
-        "arc_display_type": "Top Trump",
-        "arc_display_name": "Opel Kadett Rallye 1900",
-        "arc_description": "Opel Kadett Rallye 1900 Top Trump card",
-        "arc_serial_number": "5a",
-        "arc_home_location_identity": "locations/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
-
-        // attribute with "arc_attribute_type": "arc_attachment" is a reference to a BLOB attached to the Asset
-        "arc_primary_image": {
-          "arc_attribute_type": "arc_attachment",
-          "arc_blob_hash_value": "01ba4719c80b6fe911b091a7c05124b64eeece964e09c058ef8f9805daca546b",
-          "arc_blob_identity": "blobs/1754b920-cf20-4d7e-9d36-9ed7d479744d",
-          "arc_blob_hash_alg": "SHA256",
-          "arc_file_name": "somepic.jpeg",
-          "arc_display_name": "arc_primary_image",
-          },
-        
-        // All others are user-defined attributes
-        "bhp": "145",
-        "top_speed": "190",
-        "weight": "860",
-        "engine_capacity": "1980"
-      },
-
-      // Behaviours define the set of APIs that can be used to interact with the Asset
-      "behaviours": [
-        "RecordEvidence"
-      ],
-
-      // Date and time at which these attributes were contemporary
-      "at_time": "2021-06-25T12:40:03Z",
-
-      // Storage and integrity properties - managed by the system
-      "confirmation_status": "CONFIRMED",
-      "tracked": "TRACKED",
-      "owner": "0xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx",
-    }
-```
-
-{{< note >}}
-**Note:** The Asset Identity is fixed and constant for the life of the Asset and across all stakeholders. This ensures easy traceability and assurance that all stakeholders are collaborating on the same object and see the same complete history of Events.
-{{< /note >}}
```

Comment:
Removing the asset example, to reduce focus (space) on Assets, and the Assets API docs have all these details.